### DISCUSSION TITLE
bun js

### DISCUSSION THREAD

**Discussion by Matrex7** [pocketbase/pocketbase#3993] [Q&A] [answered]
can i contribute using bun js instead of node js +16 in the contributing page ?

**Comment by ganigeorgiev** [comment, marked as answer]
I haven't tried building the Admin UI with Bun so I'm not sure if it will work (we also use a slightly older Vite version that will be updated with the next release).

But does it really matter? The current build performance is good enough and I don't think we have a need of any of the Bun specific APIs to make it worth abandoning Node.

**Reply by aliresool621** [reply]
Bun api is simpler + https://elysiajs.com/ (even though it is a bit buggy)

**Comment by Tsiangana** [comment]
As a programmer you learn that we should work with what really matters, and not work with something just because the majority is talking about it.

In programming there are many ways to achieve a result, so as long as you alternate the result you want, I don't see any problem.

For example, I work with PHP and SQL, but at the moment I'm learning Firebase, not because I'm learning Firebase means I'm going to stop using PHP with SQL, where I find it easier to use one or the other I'm going to use

**_That's my opinion..._**

**Comment by Matrex7** [comment]
Cool cool thank you i tried contributing but i decided to make mines
